src/main/java/uk/ac/cam/cl/dtg/isaac/api/EventsFacade.java

```diff
@@ -1718,12 +1721,10 @@ private IsaacEventPageDTO augmentEventWithBookingInformation(final HttpServletRe
 
             try {
                 RegisteredUserDTO user = userManager.getCurrentRegisteredUser(request);
-                page.setUserBooked(this.bookingManager.isUserBooked(page.getId(), user.getId()));
-                page.setUserOnWaitList(this.bookingManager.hasBookingWithStatus(page.getId(), user.getId(), BookingStatus.WAITING_LIST));
                 page.setUserBookingStatus(this.bookingManager.getBookingStatus(page.getId(), user.getId()));
             } catch (NoUserLoggedInException e) {
                 // no action as we don't require the user to be logged in.
-                page.setUserBooked(null);
+                page.setUserBookingStatus(null);
```

Comment:
I assume this (and the other related changes) is OK because we don't use these properties anymore in the front-end, right?

Reply:
Ah, my commit message was very clear that I had checked this, but the PR doesn't say it. Yes, neither of these properties appear in the frontend types and they are not used by name anywhere either!